data/scenarios/02Tutorials/05-crash.yaml

```diff
@@ -0,0 +1,65 @@
+name: Debug robots
+description: |
+  Learn how to view built robots and debug them.
+entities:
+  - name: Goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - It is hard, OK?
+win: |
+  try {
+    return false;
+  } { return false }
+#solution: |
+#  crasher <- build {move; move; move; log "bye"; move};
+#  salvager <- build {move; move; move; salvage; turn back; move; move; upload base};
+robots:
+  - name: base
+    loc: [0,0]
+    dir: [1,0]
+    devices:
+      - logger
+      - 3D printer
+    inventory:
+      - [1, Goal]
+      - [10, logger]
+      - [10, compass]
+      - [10, scanner]
+  - name: secret
+    loc: [3, 0]
+    dir: [0, 0]
+    display:
+      char: '░'
+      attr: grass
+    system: true
+    program: |
+      def foreach = \s.\e.\c. if (s >= e) {} {c s; foreach (s+1) e c} end;
+      def repeat = \c. c; repeat c end;
+      repeat (foreach 1 100 (\i.
+        try {
+          r <- robotNumbered i;
+          reprogram r {log "I am going to tell you how to win this..."}
```

Comment:
@byorgey do you have a suggestion on what we could tell the base to win?

I was thinking that the secret robot could choose a random number, create that many entities (so that we could check that) and then the base would `say` that number?

Reply:
How about if there is a row of 5 locations, and the base starts with 5 objects (flower, bit, tree... something), and the secret robot picks a random permutation and tells which order to put down the items?

Reply:
I like it!